Userland/Libraries/LibTLS/Socket.cpp

```diff
@@ -67,12 +67,14 @@ bool TLSv12::write(ReadonlyBytes buffer)
         return false;
     }
 
-    PacketBuilder builder { MessageType::ApplicationData, m_context.options.version, buffer.size() };
-    builder.append(buffer);
-    auto packet = builder.build();
+    for (size_t offset = 0; offset < buffer.size(); offset += 16 * KiB) {
```

Comment:
Is `16 KiB` art of the spec? Should this be a named constant?

Reply:
18432 is part of the "spec", 16K is just a nice number smaller than that
We could make it a named constant though